pandas/core/strings.py

```diff
@@ -444,60 +442,33 @@ def str_match(arr, pat, case=True, flags=0, na=np.nan, as_indexer=False):
     flags : int, default 0 (no flags)
         re module flags, e.g. re.IGNORECASE
     na : default NaN, fill value for missing values.
-    as_indexer : False, by default, gives deprecated behavior better achieved
-        using str_extract. True return boolean indexer.
+    as_indexer : ignored
```

Comment:
Actually, maybe I should give a more specific explanation when `as_indexer=False` (the previous default behaviour) if people would have specified this explicitly, because in those cases there *is* actually a breaking change in behaviour.